pkgs/servers/nextcloud/default.nix

```diff
@@ -1,46 +1,60 @@
-{ lib, stdenvNoCC, fetchurl, nixosTests
-, nextcloud28Packages
-, nextcloud29Packages
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  nixosTests,
+  nextcloud28Packages,
+  nextcloud29Packages,
+  nextcloud30Packages,
 }:
 
 let
-  generic = {
-    version, hash
-  , eol ? false, extraVulnerabilities ? []
-  , packages
-  }: stdenvNoCC.mkDerivation rec {
-    pname = "nextcloud";
-    inherit version;
-
-    src = fetchurl {
-      url = "https://download.nextcloud.com/server/releases/${pname}-${version}.tar.bz2";
-      inherit hash;
-    };
+  generic =
+    {
+      version,
+      hash,
+      eol ? false,
+      extraVulnerabilities ? [ ],
+      packages,
+    }:
+    stdenvNoCC.mkDerivation rec {
+      pname = "nextcloud";
+      inherit version;
 
-    passthru = {
-      tests = nixosTests.nextcloud;
-      inherit packages;
-    };
+      src = fetchurl {
+        url = "https://download.nextcloud.com/server/releases/${pname}-${version}.tar.bz2";
+        inherit hash;
+      };
+
+      passthru = {
+        tests = nixosTests.nextcloud;
```

Comment:
It would be nice to have this only be the tests with the appropriate major version suffix, but I can make that change in a separate PR.

Reply:
https://github.com/NixOS/nixpkgs/pull/342896